### PR TITLE
Move scanning behind "scan" command

### PR DESCRIPTION
**Pull request by Piccirello**
Also adds "list" command

### CODE REVIEW

File: src/cli.js [hunk 1]
```diff
@@ -84,34 +123,79 @@ class CLI {
...
+    .description("Print radar configuration")
+    .option("--json", "Output results as json blob")
+    .action((listType, options) => {
+      cli.setOptions(options);
+      cli.list(listType);
+    });
+}
+
+const cli = new CLI();
+loadCommands();
```
Comment (bvallelunga) [inline src/cli.js]: Why is loadCommands a function, looks like it is only called once?

File: src/objects/file.js [hunk 2]
```diff
@@ -9,22 +11,27 @@ class File {
...
+    return this._fullPath;
   }
 
   size() {
     return this._size;
   }
 
   extension() {
-    if (this._extension == null) {
+    if (this._extension === null) {
```
Comment (bvallelunga) [inline src/objects/file.js]: +1

**Comment by bvallelunga** [comment]
Can we update the ReadMe with the new commands?

**Comment by Piccirello** [comment]
I’ll update the readme before merging this

File: src/cli.js (hunk 1 above)
Comment (Piccirello) [reply src/cli.js]: I’m using the function as a way of organizing/documenting the code, rather than as a way of making the code reusable.

Comment (bvallelunga) [reply src/cli.js]: Gotcha

**Pull request closed without merging**